theme.json

```diff
@@ -48,8 +48,54 @@
 			]
 		},
 		"layout": {
-			"contentSize": "840px",
-			"wideSize": "1100px"
+			"contentSize": "645px",
+			"wideSize": "1340px"
+		},
+		"spacing": {
+			"defaultSpacingSizes": false,
+			"spacingSizes": [
+				{
+					"name": "X-Tiny",
+					"size": "5px",
+					"slug": "10"
```

Comment:
For better code readability and maintainability, we should use descriptive slugs such as 'x-tiny', 'tiny', 'small', 'regular', 'large', 'x-large', and 'xx-large'

Reply:
Except for the slug "10", these slugs are the defaults and they are chosen because any other slugs will make it more difficult to switch themes.

If one theme uses the slug "tiny" and the other theme uses "20", then the spacing will break -it will be missing- when switching between them.

If the slugs are the same, then the spacing will be different because the value is different, but it wont be missing.

Reply:
I understand, thank you for the clarification.